EvmYul/EVM/Semantics.lean

```diff
@@ -955,9 +968,7 @@ def Lambda
             | .none => false
         if debugMode ∧ F₀ then
           dbg_trace "Contract creation failed: account {toHex (BE a)} already existed."
-        let F₁ : Bool := σStarStar == ∅
-        if debugMode ∧ F₁ then
-          dbg_trace "Contract creation failed: the code execution failed."
+        -- let F₁ : Bool := σStarStar == ∅ ∧ some returnedData == .none
```

Comment:
Not sure why this comment stayed, this is just a roundabout way to say let `F1 := false`, because .some is never .none (injectivity of constructors).